crates/virtio-queue/src/chain.rs

```diff
@@ -32,7 +32,7 @@ pub struct DescriptorChain<M> {
     is_indirect: bool,
 }
 
-impl<M> DescriptorChain<M>
+impl<'a, M> DescriptorChain<M>
```

Comment:
I would move the lifetime down to the two functions that require it. Or does anything require it on this top-level?

Reply:
I'll move it to the two functions that require it.